docs/developer/address.mdx

```diff
@@ -177,3 +177,14 @@ Each field is validated separately, and each issue has a corresponding error cod
 
 - `postalCode` with the wrong format throws the `INVALID` error
 - missing fields mentioned in `requiredFields` throw the `REQUIRED` error
+
+## Skipping the address validation
+
+Since version 3.19, Saleor may completely skip the address validation. To do so, the `AddressInput` must receive `skipValidation` flag set to `true`.
+This will turn off whole validation, except `country` field - this one requires proper `CountryCode` enum. Please note, that to use `skipValidation` flag,
+Saleor may demand some extra permissions. Also, not all mutations support this feature. For more info, please check specific mutation details.
```

Comment:
can we somewhere attach information regarding required permission for skipping validation?